sdcm/cluster_aws.py

```diff
@@ -522,7 +522,7 @@ def check_spot_termination(self):
 
     @property
     def is_data_device_lost_after_reboot(self) -> bool:
-        if self.parent_cluster.is_data_volume_on_ebs():
+        if self.parent_cluster.params.get("data_volume_disk_num") > 0:
```

Comment:
It would be better to keep the original code and implement this method - so we keep intention - `data_volume_disk_num` param is not showing it.